hnn_core/dipole.py

```diff
@@ -102,9 +102,13 @@ def average_dipoles(dpls):
         average over the same components in the input list
     """
     # need at least one Dipole to get times
-    if len(dpls) < 2:
-        raise ValueError("Need at least two dipole object to compute an"
-                         " average")
+    if len(dpls) == 1:
```

Comment:
I think this function shouldn't throw any warnings if:

1) You get a list (with len > 0)
2) Each element of the list is a `Dipole` object

Also, if you really want to throw a warning, you should use the `warnings` module:

```py
from warnings import warn

warn('important message')
```

Reply:
@jasmainak This is me just nitpicking but I was thinking that it might help to let users know that when calling the average_dipoles() function when only one trial was run, they are effectively not averaging anything. I'm cool with removing the message if you think it doesn't help much.

Reply:
I also prefer to limit the number of warnings and only throw them when there's something to be _careful_ about. I think `average` should work silently for a single instance, makes it more flexible.

Reply:
Alrighty, I'll remove the message :)